### PR TITLE
Add configuration option for print header

### PR DESCRIPTION
**Pull request by vuilleumierc**
Add new flag in printable extract to be able to hide canton logo from PDF header (GEO-4197)

### CODE REVIEW

File: pyramid_oereb/contrib/print_proxy/mapfish_print.py [hunk 1]
```diff
@@ -544,6 +544,8 @@ def convert_to_printable_extract(self, extract_dict, feature_geometry, pdf_to_jo
                 if 'NrOfPoints' in legend:
                     legend['NrOfPoints'] = '{0}'.format(legend['NrOfPoints'])
 
+        extract_dict['PrintCantonLogo'] = Config.get('print_canton_logo')
```
Comment (jwkaltz) [inline pyramid_oereb/contrib/print_proxy/mapfish_print.py:547]: Default must be true (if the Config item is missing)

File: pyramid_oereb/standard/pyramid_oereb.yml.mako [hunk 2]
```diff
@@ -248,6 +248,9 @@ pyramid_oereb:
   # The method used to return the municipality logos.
   get_municipality_method: pyramid_oereb.standard.hook_methods.get_municipality
 
+  # Flag to print or not the canton logo
```
Comment (jwkaltz) [inline pyramid_oereb/standard/pyramid_oereb.yml.mako:251]: This must be in the print configuration section (only for mapfish print configuration options)

**Comment by voisardf** [comment]
sorry to intervene. Who asked for this configuration, what is exactely the reason behind it? As I believe to recall, the logos are mandatory in the specification.

**Comment by jwkaltz** [comment]
> sorry to intervene. Who asked for this configuration, what is exactely the reason behind it? As I believe to recall, the logos are mandatory in the specification. - or is it just dependent on the print method used?

Hi @voisardf , indeed the cantonal logs will still be active by default. We want to add a configuration option for Liechtenstein, who wants to be able to not use a cantonal logo, but still stick to the standard code and print templates.

**Comment by voisardf** [comment]
which makes perfect sense! My apologies to Lichtenstein 😃 👍

**Review by jwkaltz — APPROVED** [review]

**Pull request merged**